app/views/categories/show.html.erb

```diff
@@ -0,0 +1,23 @@
+<p style="color: green"><%= notice %></p>
+
+<div class="container">
+
+  <div class="shadow-sm category-details d-flex gap-3">
+    <div class="icon-img">
+      <img class="category-img" src=<%=@category.icon%> alt<%=@category.name%> />
+    </div >
+    <div class="category-text text-secondary gap-2 d-flex flex-column align-self-end">
+      <div class="category-name border-bottom d-flex justify-content-between align-items-center">
+        <span class="fw-bold fs-4"><%= @category.name %></span>
+        <span class="fw-bold fs-4">$<%= @total_expenses %></span>
+        <div class="d-flex gap-2 align-items-baseline justify-content-center mb-2">
+          <%= link_to "EDIT", edit_category_path(@category), class: "cat-command", id: "edit" %>
+          <%= button_to "DELETE", category_path(@category), method: :delete, class: "cat-command", id: "delete" %> 
+        </div>
+      </div>
+      <span class="time fst-italic"><%= @category.created_at.strftime('%d %B %Y at %I:%M %p') %></span>
+    </div>
```

Comment:
- [ ] When I click on a specific category that has transactions, I expect to see the transactions in that particular category listed on the page. 
In the attached image, you can see that I have done a few transactions but they are not listed. I think you missed this particular feature. Kindly implement it 🙏 
![image](https://user-images.githubusercontent.com/46839548/212472823-e8b041d6-f8de-4fd2-908f-a2bec7e08d2b.png)
![image](https://user-images.githubusercontent.com/46839548/212472940-b82fbe92-fb76-4825-bf03-af825b6faece.png)